js/src/ui/AccountCard/accountCard.js

```diff
@@ -0,0 +1,190 @@
+    account: PropTypes.object.isRequired,
+    onClick: PropTypes.func.isRequired,
+    onFocus: PropTypes.func.isRequired,
+
+    balance: PropTypes.object
+  };
+
+  state = {
+    copied: false
+  };
+
+  render () {
+    const { account } = this.props;
+    const { copied } = this.state;
+
+    const { address, name, meta = {} } = account;
+
+    const displayName = (name && name.toUpperCase()) || address;
+    const { tags = [] } = meta;
+
+    const classes = [ styles.account ];
+
+    if (copied) {
+      classes.push(styles.copied);
+    }
+
+    return (
+      <div
+        key={ address }
+        tabIndex={ 0 }
+        className={ classes.join(' ') }
+        onClick={ this.onClick }
+        onFocus={ this.onFocus }
+        onKeyDown={ this.handleKeyDown }
+      >
+        <IdentityIcon address={ address } />
+        <div className={ styles.accountInfo }>
+          <div className={ styles.accountName }>
+            <span>{ displayName }</span>
+          </div>
+
+          { this.renderTags(tags, address) }
+          { this.renderAddress(displayName, address) }
+          { this.renderBalance(address) }
+        </div>
+      </div>
+    );
+  }
+
+  renderAddress (name, address) {
+    if (name === address) {
+      return null;
+    }
+
+    return (
+      <div className={ styles.addressContainer }>
+        <span
+          className={ styles.address }
+          onClick={ this.preventEvent }
+          ref={ `address` }
+          title={ address }
+        >
+          { address }
+        </span>
+      </div>
+    );
+  }
+
+  renderTags (tags = [], address) {
+    if (tags.length === 0) {
+      return null;
+    }
+
+    return (
+      <Tags tags={ tags } />
+    );
+  }
+
+  renderBalance (address) {
+    const { balance = {} } = this.props;
+
+    if (!balance.tokens) {
+      return null;
+    }
+
+    const ethToken = balance.tokens
+      .find((tok) => tok.token && (tok.token.tag || '').toLowerCase() === 'eth');
+
+    if (!ethToken) {
+      return null;
+    }
+
+    const value = fromWei(ethToken.value).toFormat(3);
+
+    return (
+      <div className={ styles.balance }>
+        <span>{ value }</span>
+        <span className={ styles.tag }>ETH</span>
+      </div>
+    );
+  }
+
+  handleKeyDown = (event) => {
+    const codeName = keycode(event);
+
+    if (event.ctrlKey) {
+      // Copy the selected address if nothing selected and there is
+      // a focused item
+      const isSelection = !window.getSelection || window.getSelection().type === 'Range';
+
+      if (codeName === 'c' && !isSelection) {
+        const element = ReactDOM.findDOMNode(this.refs.address);
+
+        // Copy the address from the right element
+        // @see https://developers.google.com/web/updates/2015/04/cut-and-copy-commands
+        try {
+          const range = document.createRange();
+          range.selectNode(element);
+          window.getSelection().addRange(range);
+          document.execCommand('copy');
```

Comment:
You may want to consider [all the quirks involved with the clipboard](https://github.com/lgarron/clipboard.js/blob/23c6270/clipboard.js). Haven't found a focused & well-written module yet though.